js/Source/Request/Behavior.PollForUpdate.js

```diff
@@ -30,6 +34,7 @@ provides: [Behavior.PollForUpdate]
         url: api.get('url'),
         date: api.get('date'),
         pollInterval: api.get('pollInterval') || 60000,
+        haltOnFailure: api.get('haltOnFailure')
```

Comment:
`.getAs(Boolean, 'haltOnFailure')`

will throw you an error if its something else.